symphony/lib/core/class.symphony.php

```diff
@@ -93,14 +93,6 @@ protected function __construct()
             General::cleanArray($_POST);
         }
 
-        // Set date format throughout the system
-        define_safe('__SYM_DATE_FORMAT__', self::Configuration()->get('date_format', 'region'));
-        define_safe('__SYM_TIME_FORMAT__', self::Configuration()->get('time_format', 'region'));
-        define_safe('__SYM_DATETIME_FORMAT__', __SYM_DATE_FORMAT__ . self::Configuration()->get('datetime_separator', 'region') . __SYM_TIME_FORMAT__);
-        DateTimeObj::setSettings(self::Configuration()->get('region'));
-
-        self::initialiseErrorHandler();
```

Comment:
The error handler is already initialised in the [bundle.php](https://github.com/symphonycms/symphony-2/blob/master/symphony/lib/boot/bundle.php#L27), therefore this is initialising it again :)